ui/components/component-library/text/text.scss

```diff
@@ -27,7 +25,7 @@ $text-variants: (
 
 .mm-text {
   // Set default styles
-  font-family: var(--font-family-sans);
+  font-family: var(--font-family-default);
```

Comment:
Replacing removed CSS variables as per the [migration guide](https://github.com/MetaMask/metamask-design-system/blob/main/packages/design-tokens/MIGRATION.md#from-version-600-to-700)